docs/en/compute/Compute-Presto.md

```diff
@@ -9,8 +9,6 @@ priority: 2
 * Table of Contents
 {:toc}
 
-## Overview
```

Comment:
Name it as `Prerequisites`?

Reply:
see line 25 for `Prerequisites` . this is plain intro